src/FilenameParsing/FileIDHelper.php

```diff
@@ -11,7 +11,7 @@ interface FileIDHelper
      * The string that separates the base file name from the variant name.
      * This string must not be present in the name of files which are not variants.
      */
-    public const VARIANT_SEPARATOR = '__';
+    public const string VARIANT_SEPARATOR = '__';
```

Comment:
This should have been done in https://github.com/silverstripe/silverstripe-assets/pull/687
It's supported as of PHP 8.3 so should be the way we always add constants from now on.